02-props-state-component-architecture/props/Props Homework/src/Pokedex.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import './Pokedex.css';
+import Pokecard from './Pokecard'
+import { Grid, Col, Row } from 'react-bootstrap';
+
+class Pokedex extends Component {
+  render() {
+    const pokemon = this.props.pokemon.map(pokemon => {
+      return (
+        <Col s={6} md={4} key = {pokemon.id}>
+          <Pokecard
+            name = {pokemon.name}
+            type = {pokemon.type}
+            image = {pokemon.image}
```

Comment:
this is where you need an ID, since it's within a `map`. Something like this works:

```jsx
const { id, ...rest } = pokemon;
<Pokecard
   key={id}
   {...rest}
 />
```